src/doc/documentation-guide.md

```diff
@@ -94,6 +95,72 @@ that concatenates the various `README.md` files we have.
 
 # Stylistic Conventions
 
+## General Considerations
+
+Documentation is most effective when it is precise, concise and uniform.
+Documentation is not a novel, or a blog post. It should not be fun to read.
+Documentation should be normative, and as such be an especially unsurprising
```

Comment:
@wingo While the notation you used serves as an example this is in no way personally about you or the documentation you wrote (other people wrote documentation too, even before last week). Could we agree on not taking these kind of things to a personal level? I find this extremely annoying. If I have a personal issue I will contact you personally in private.

> It feels like more of a blog post than documentation ;) Or rather: too much opining for the guidelining.

Absolutely agree.